FeedAPIChallenge/Feed API/FeedImageItemMapper.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  FeedImageItemMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Guilherme Sol on 11/09/2021.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+internal class FeedImageItemMapper {
+	
+	private static var OK_200:Int{ 200 }
+	
+	private struct Root: Decodable {
+		var items: [FeedImageItem]
+		var feedItems: [FeedImage] {
+			items.compactMap({ $0.feedImage })
+		}
+	}
+
+	private struct FeedImageItem: Decodable {
+		var image_id: String
+		var image_desc: String?
+		var image_loc: String?
+		var image_url: URL
```

Comment:
As the properties on these models are only set one, It would be more suitable to have them declared as constants.

```suggestion
		let items: [FeedImageItem]
		var feedItems: [FeedImage] {
			items.compactMap({ $0.feedImage })
		}
	}

	private struct FeedImageItem: Decodable {
		let image_id: String
		let image_desc: String?
		let image_loc: String?
		let image_url: URL
```

This makes it clear to the next developer, these values should not be mutated once set.